README.md

```diff
@@ -53,6 +53,41 @@ that you might have to run sometimes.
  - `git-fatlas-remake-package-list`: force rebuilding the package
    list. If you add or remove a bunch of things you might have to call
    this. Hopefully not though.
+   
+  
+Working with tags, merge requests or branches which are not `master`
+-------------------------------------------------------------------
+
+Running 
+```
+git-fatlas-init
```

Comment:
this will currently check out `21.2`, not `master`. If you want to change the default branch to `master` I'm fine with that though.

Reply:
I guess more and more people is moving on working on `master`, so since you are fine with this I'm changing the default release